samples/web/send-from-aspnetcore-webapi/Core_6/WebApplication/WebApplication.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net461</TargetFramework>
+    <RuntimeIdentifier>win7-x86</RuntimeIdentifier>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Folder Include="wwwroot\" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore" Version="1.1.1" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="1.1.2" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.1.1" />
+    <PackageReference Include="NServiceBus" Version="6.2.1" />
+  </ItemGroup>
```

Comment:
It seems to be added transitively by:

```xml
<PackageReference Include="Microsoft.AspNetCore" Version="1.1.1" />
```

We still have a call to `UseKestrel()` in the as well as `UseIISIntegration()` so nothing has changed here.